pkg/conf/override.go

```diff
@@ -54,5 +56,16 @@ func (config *NodeConfig) ToOverrideEnv(host string, nc NetworkConfig) map[strin
 		overrideEnv[k] = v
 	}
 
+	// register plugins and pass their configs into
+	// override .env
+	plugins := []string{}
```

Comment:
What's the difference between `[]string{}` and `[]string`?

Reply:
Resolved, for some reason I thought this was a declaration and not an assignment making it an empty slice literal.

Reply:
Yep, i could've done `var plugins []string` and i think got the same result. Sometimes default go types I forget how they work so I do the assignment as it's a bit more predictable.

Reply:
I'm not sure which one is exactly more idiomatic but both work